### ISSUE TITLE
[css-grid] Include scrollbar gutter in #subgrid-edge-placeholders

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#9935]
https://drafts.csswg.org/css-grid/#subgrid-item-contribution

> In this process, the sum of the [subgrid](https://drafts.csswg.org/css-grid/#subgrid)’s **margin, padding, [scrollbar gutter](https://drafts.csswg.org/css-overflow-3/#scrollbar-gutter), and border** at each edge

https://drafts.csswg.org/css-grid/#subgrid-edge-placeholders

> For each edge of a non-empty [subgrid](https://drafts.csswg.org/css-grid/#subgrid), to account for the subgrid’s **margin/border/padding** at that edge [...]

The 2nd quote doesn't include the [scrollbar gutter](https://drafts.csswg.org/css-overflow-3/#scrollbar-gutter), but it should, like the 1st one. As per #9815, browsers are including it.

**Comment by frivoal** [comment]
Yes. It's between the padding and the border, so anything else would seem very strange.
